lib/SPIRV/SPIRVLowerSPIRBlocks.cpp

```diff
@@ -45,6 +45,7 @@
 #include "llvm/Analysis/AliasAnalysis.h"
 #include "llvm/Analysis/AssumptionCache.h"
 #include "llvm/Analysis/CallGraph.h"
+#include "llvm/InitializePasses.h"
```

Comment:
Please move this after `#include "llvm/IR/Verifier.h"`

Reply:
Done